### DISCUSSION TITLE
WiX Online Meeting #219

### DISCUSSION THREAD

**Discussion by robmen** [wixtoolset/issues#6570] [Announcements]
The next WiX Online meeting will be: September 9th, 2021 @9:30 Pacific

Agenda items:

* Triage
* Design Discussions
  * #5950 – Canceled ExePackage rollback behavior
  * .NET Framework signature changes

Join us live: https://www.twitch.tv/firegiantco/
Join us later: https://www.youtube.com/c/wixtoolset/

**Comment by ericstj** [comment]
May I ask that you add https://github.com/wixtoolset/issues/issues/6556 to the agenda?  Thanks!

**Reply by rseanhall** [reply]
The Triage bullet covers that, we go over all issues in this query: https://github.com/wixtoolset/issues/issues?q=is%3Aissue+label%3Atriage
